### DISCUSSION TITLE
Will Oxygen Support Hosting Browser Pages?

### DISCUSSION THREAD

**Discussion by TheCedarPrince** [OxygenFramework/Oxygen.jl#56] [General]
Hi @ndortega , 

I have been greatly curious about Oxygen.jl for quite some time as it seems fantastic! Saw your post on Forem and ever since then, have been thinking about it. 

A question that I had is if Oxygen will ever support things like hosting a web pages and web page components? I am a huge fan of the [GenieFramework](https://github.com/GenieFramework) and their [Stipple](https://github.com/GenieFramework/Stipple.jl) package is one of my favorite tools for fast dashboard prototyping. I am curious if Oxygen will ever similarly support web pages or if it will ever seek to interoperate with tooling such as Stipple? Oxygen seems to me like a great way to host and route multiple Stipple apps at once to create a multi-page website as well as other sort of pages for easy hosting.

Obviously this may be outside the scope of Oxygen but I would be foolish to not say that I see a lot of great opportunities with Oxygen! Nonetheless, great work! 

Thanks!

~ tcp :deciduous_tree:

**Comment by ndortega** [comment]
Hi @TheCedarPrince , 

Thanks for the kind words! You're correct in assuming that integrating with stipple was never part of my mental road map for this project. But that doesn't mean it's completely out of the question. I remember seeing at least one other person bring up this same point in the comment section of that Forem post, so there have to be at least dozens of other people who feel the same way. 

Admittedly, I've never used Stipple.jl but after taking a quick look through the repo it appears to be pretty tightly coupled to Genie.jl package. I can't promise I'll be able to integrate with Stipple, but I do enjoy the challenge of hacking around and making code do things it shouldn't. So I'll make this promise instead: I'll give it an honest try and see what comes of it.

Thanks for the post!

~ Nate
